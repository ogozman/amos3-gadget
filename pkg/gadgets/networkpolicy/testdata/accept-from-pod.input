[
  {
    "type": "accept",
    "remote_kind": "pod",
    "port": 50051,
    "local_pod_namespace": "demo",
    "local_pod_name": "shippingservice-778db496dd-v94bv",
    "local_pod_owner": "shippingservice",
    "local_pod_labels": {
      "app": "shippingservice",
      "pod-template-hash": "778db496dd"
    },
    "remote_pod_namespace": "demo",
    "remote_pod_name": "checkoutservice-66d68cbdd-9kkkl",
    "remote_pod_labels": {
      "app": "checkoutservice",
      "pod-template-hash": "66d68cbdd"
    },
    "debug": "101619576114727 cpu#1 accept 15451 shippingservice 10.2.187.31:50051 10.2.187.25:58730 4026532885\n"
  }
]
