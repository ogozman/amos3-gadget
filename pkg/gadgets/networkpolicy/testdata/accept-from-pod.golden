apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  creationTimestamp: null
  name: shippingservice-network
  namespace: demo
spec:
  ingress:
  - from:
    - podSelector:
        matchLabels:
          app: checkoutservice
    ports:
    - port: 50051
      protocol: TCP
  podSelector:
    matchLabels:
      app: shippingservice
  policyTypes:
  - Ingress
  - Egress
